anda/misc/tracy/tracy.spec

```diff
@@ -0,0 +1,65 @@
+Name:			tracy
+Version:		0.10
+Release:		1%?dist
+Summary:		A real time, nanosecond resolution, remote telemetry, hybrid frame and sampling profiler for games and other applications.
+License:		BSD-3-Clause
+URL:			https://github.com/wolfpld/tracy
+Source0:		https://github.com/wolfpld/tracy/archive/refs/tags/v%version.tar.gz
+BuildRequires:  cmake meson gcc libxkbcommon dbus-devel libglvnd glfw-devel freetype-devel pkgconfig(capstone) pkgconfig(libunwind) pkgconfig(libdebuginfod) pkgconfig(tbb)
+Patch:          https://github.com/wolfpld/tracy/commit/1a971d867d6fa5bf6dc57d705dcbbc6020031e7a.patch
+
+%description
+%A real time, nanosecond resolution, remote telemetry, hybrid frame and sampling profiler for games and other applications.
+
+%prep
+%autosetup -n tracy-0.10
+
+%build
+%make_build -C capture/build/unix release
+%make_build -C csvexport/build/unix release
+%make_build -C import-chrome/build/unix release
+%make_build -C library/unix release
+%make_build -C profiler/build/unix release
+%make_build -C update/build/unix release
+
+%install
+mkdir -p %{buildroot}/usr/bin/
+install -m 755 Tracy-release %{buildroot}/usr/bin/Tracy-release
+
+%files
+
+%license LICENSE
+%doc README.*
+%{_datadir}/tracy/capture
```

Comment:
instead of listing out all the files, include the tracy directory itself

Reply:
So instead of listing out all of these, I just do:

```
%files

/tracy
```

?

Or listing the LICENSE and README.*, then /Tracy?

Reply:
`%_datadir/tracy` alongside the licenses and stuff